halo/halo.py

```diff
@@ -14,7 +14,8 @@
 from spinners.spinners import Spinners
 from log_symbols.symbols import LogSymbols
 
-from halo._utils import is_supported, colored_frame, is_text_type, decode_utf_8_text, get_terminal_columns, \
+from halo import _utils
```

Comment:
Use `halo._utils` to import all utils or revert this change. I don't see the point of importing this separately.